src/Makefile.am

```diff
@@ -38,6 +38,7 @@ libmodplug_la_SOURCES = tables.h         \
                         load_j2b.cpp       \
                         load_mt2.cpp       \
                         load_psm.cpp       \
+                        load_gdm.cpp       \
```

Comment:
New source needs adding to `CMakeLists.txt` too

Reply:
OK, done.